packages/ui/v2/modules/event-types/CreateEventType.tsx

```diff
@@ -202,7 +204,15 @@ export default function CreateEventTypeButton(props: CreateEventTypeBtnProps) {
                 value={teamId}
               />
             )}
-            <TextField label={t("title")} placeholder={t("quick_chat")} {...register("title")} />
+            <TextField
+              label={t("title")}
+              placeholder={t("quick_chat")}
+              {...register("title")}
+              onChange={(e) => {
+                form.setValue("title", e?.target.value);
+                setTitleCopy(e?.target.value);
+              }}
+            />
```

Comment:
why do we need to do this here too?

Reply:
This was here because in the first approach the updated value of title was needed. But i could have used watch. So this step was unnecessary.